model/infinispan/src/main/java/org/keycloak/models/sessions/infinispan/UserSessionAdapter.java

```diff
@@ -91,7 +91,7 @@ public Map<String, AuthenticatedClientSessionModel> getAuthenticatedClientSessio
                 // Check if client still exists
                 ClientModel client = realm.getClientById(key);
                 if (client != null) {
-                    final AuthenticatedClientSessionAdapter clientSession = provider.getClientSession(this, client, value, offline);
+                    final AuthenticatedClientSessionModel clientSession = provider.getClientSession(this, client, value, offline);
```

Comment:
Why is this change here? The `InfinispanUserSessionProvider` still returns `AuthenticatedClientSessionAdapter`, so this change seems redundant.

Reply:
Because previously this called directly method `public AuthenticatedClientSessionAdapter getClientSession(UserSessionModel userSession, ClientModel client, UUID clientSessionId, boolean offline)` in `InfinispanUserSessionProvider` while now it calls `default AuthenticatedClientSessionModel getClientSession(UserSessionModel userSession, ClientModel client, UUID clientSessionId, boolean offline)` in `UserSessionProvider`. 
Btw, newly called method is deprecated so maybe I should call rather `getClientSession(UserSessionModel userSession, ClientModel client, String clientSessionId, boolean offline);` ?